mozdownload/scraper.py

```diff
@@ -36,7 +36,10 @@
 APPLICATIONS = ('b2g', 'firefox', 'thunderbird')
 
 # Base URL for the path to all builds
-BASE_URL = 'https://ftp.mozilla.org/pub/mozilla.org'
+BASE_URL = 'http://download.cdn.mozilla.net/pub'
+
+# In case CDN fails
+BACKUP_URL = 'https://ftp.mozilla.org/pub/mozilla.org'
```

Comment:
We don't only support installer files, but any kind of file which can exist inside a build folder. This included also the packaged tests, the zip build for windows for nightly builds, etc. I had a look at the given URL and was forwarded to ftp.m.o in case for nightly, tinderbox, and try builds. I can see all the files we need even including the mar files. So I'm not sure what you meant that mar files will not be present. Does your comment only apply to candidates and releases? Otherwise it looks good and good to see that there is a way to retrieve files via HTTPS.

Nick what I wonder is if connections are going outside of ftp.m.o to the CDN network for at least the release builds. When I navigate through the folder structure and trying to download various installers, always ftp.mo is used as backend server.